notebooks/00_quick_start/xdeepfm_movielens.ipynb

```diff
@@ -0,0 +1,1541 @@
+       "      <th>99999</th>\n",
+       "      <td>3.0</td>\n",
+       "      <td>1:89:1</td>\n",
+       "      <td>2:2625:1</td>\n",
+       "      <td>3:1:0</td>\n",
+       "      <td>3:2:0</td>\n",
+       "      <td>3:3:0</td>\n",
+       "      <td>3:4:0</td>\n",
+       "      <td>3:5:0</td>\n",
+       "      <td>3:6:0</td>\n",
+       "      <td>3:7:1</td>\n",
+       "      <td>...</td>\n",
+       "      <td>3:10:0</td>\n",
+       "      <td>3:11:0</td>\n",
+       "      <td>3:12:0</td>\n",
+       "      <td>3:13:0</td>\n",
+       "      <td>3:14:0</td>\n",
+       "      <td>3:15:0</td>\n",
+       "      <td>3:16:0</td>\n",
+       "      <td>3:17:0</td>\n",
+       "      <td>3:18:0</td>\n",
+       "      <td>3:19:0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>5 rows × 22 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "       rating   userID    itemID genre1 genre2 genre3 genre4 genre5 genre6  \\\n",
+       "99995     4.0  1:505:1  2:2621:1  3:1:0  3:2:0  3:3:0  3:4:0  3:5:0  3:6:0   \n",
+       "99996     3.0   1:89:1  2:2622:1  3:1:0  3:2:0  3:3:0  3:4:0  3:5:0  3:6:0   \n",
+       "99997     3.0   1:89:1  2:2623:1  3:1:0  3:2:0  3:3:0  3:4:0  3:5:0  3:6:0   \n",
+       "99998     3.0   1:89:1  2:2624:1  3:1:0  3:2:0  3:3:0  3:4:0  3:5:0  3:6:0   \n",
+       "99999     3.0   1:89:1  2:2625:1  3:1:0  3:2:0  3:3:0  3:4:0  3:5:0  3:6:0   \n",
+       "\n",
+       "      genre7   ...   genre10 genre11 genre12 genre13 genre14 genre15 genre16  \\\n",
+       "99995  3:7:0   ...    3:10:0  3:11:0  3:12:0  3:13:0  3:14:0  3:15:0  3:16:0   \n",
+       "99996  3:7:0   ...    3:10:0  3:11:0  3:12:0  3:13:0  3:14:0  3:15:0  3:16:0   \n",
+       "99997  3:7:0   ...    3:10:0  3:11:0  3:12:0  3:13:0  3:14:0  3:15:0  3:16:0   \n",
+       "99998  3:7:0   ...    3:10:0  3:11:0  3:12:0  3:13:0  3:14:0  3:15:0  3:16:0   \n",
+       "99999  3:7:1   ...    3:10:0  3:11:0  3:12:0  3:13:0  3:14:0  3:15:0  3:16:0   \n",
+       "\n",
+       "      genre17 genre18 genre19  \n",
+       "99995  3:17:0  3:18:0  3:19:0  \n",
+       "99996  3:17:0  3:18:0  3:19:0  \n",
+       "99997  3:17:0  3:18:0  3:19:0  \n",
+       "99998  3:17:0  3:18:0  3:19:0  \n",
+       "99999  3:17:0  3:18:0  3:19:0  \n",
+       "\n",
+       "[5 rows x 22 columns]"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data_transformed.tail()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Parameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download data\n",
+    "xDeepFM uses the FFM format as data input: `<label> <field_id>:<feature_id>:<feature_value>`  \n",
+    "Each line represents an instance, `<label>` is a binary value with 1 meaning positive instance and 0 meaning negative instance. \n",
```

Comment:
It looks like the problem in this notebook is to predict ratings. Then the labels are numerical instead of binary values.